indico/modules/events/contributions/models/fields.py

```diff
@@ -99,14 +99,13 @@ class ContributionField(db.Model):
     # - abstract_values (AbstractFieldValue.contribution_field)
     # - contribution_values (ContributionFieldValue.contribution_field)
 
-    @property
-    def field(self):
+    def get_field(self, management=False):
```

Comment:
I wonder if it'd make more sense to have `field` and `mgmt_field` (or `management_field`, but that's a bit long ;)) properties instead (which then call `_get_field(management=True|False)`)